tests/sos_tests.py

```diff
@@ -313,7 +313,9 @@ def assertFileCollected(self, fname):
         :param fname:  The name of the file within the archive
         :type fname:  ``str``
         """
-        if os.path.exists(fname):
+        if fname.startswith(('sos_', '/sos_')):
+            self.assertFileExists(self.get_name_in_archive(fname))
+        elif os.path.exists(fname):
             self.assertFileExists(self.get_name_in_archive(fname))
```

Comment:
Can't we simplify this to:

    if fname.startswith(('sos_', '/sos_')) or os.path.exists(fname):
        self.assertFileExists(self.get_name_in_archive(fname))
    else:
        assert True

?